src/main/java/com/gotocompany/depot/metrics/MaxComputeMetrics.java

```diff
@@ -0,0 +1,20 @@
+package com.gotocompany.depot.metrics;
+
+import com.gotocompany.depot.config.SinkConfig;
+
+public class MaxComputeMetrics extends SinkMetrics {
```

Comment:
we should capture metrics for the schema issue?

Reply:
this will be catered by @Vaishnavi190900